### PR TITLE
Hint about new default node js version in buildpack

### PR DESCRIPTION
**Pull request by fwilhe**
Hi, it looks like the version info here is outdated.

Is it still required to suggest people to use node 14 by default?

See https://github.com/cloudfoundry/nodejs-buildpack/releases/tag/v1.8.4 for release notes of the buildpack.

### CODE REVIEW

**Comment by DavidMitreski5** [comment]
Thank you for your feedback! We’ll look into it and come back to you if we have any questions.

**Comment by Joysie** [comment]
Hi @fwilhe,

Unfortunately, Node 14 is **still** the version that is currently recommended.  The reason: an incompatibility issue with the _@sap/xsjs_ library that breaks all XSA applications.  See: [Application stopped working after upgrade to Node v.16](https://ga.support.sap.com/dtp/viewer/#/tree/3254/actions/51226:51218:52922/?version=current)

The developer team is working on  a solution so that Node 16 and latest versions to be supported by the _node_buildpack_ soon.

**Comment by fwilhe** [comment]
Hi @Joysie, @DavidMitreski5 

my pr does **not change the recommendation**, it only adds the correct information that node 18 is now the default in the cf buildpack. Please consider merging the pr to keep users informed about the cf buildpack.

But also: node 14 is eol in april this year https://nodejs.dev/en/about/releases/
Libraries which still require using node 14 should not be recommended to be used.

Best,
Florian

File: docs/30-development/developing-node-js-in-the-cloud-foundry-environment-3a7a0be.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The SAP `nodejs_buildpack` supports the following versions:
 
 -   Node.js **14** – recommended version
 -   Node.js **16** – supported but not recommended to use, due to current issues with the *@sap/xsjs* library. Therefore, if you use this library, bear in mind that it will be incompatible with the Node.js 16 runtime.
-
+-   Node.js **18** – default version starting with [version 1.8.4 of the buildpack](https://github.com/cloudfoundry/nodejs-buildpack/releases/tag/v1.8.4)
```
Comment (marikaner) [inline docs/30-development/developing-node-js-in-the-cloud-foundry-environment-3a7a0be.md:60]:
```suggestion
-   Node.js **18** – default version starting with [version 1.8.4 of the buildpack](https://github.com/cloudfoundry/nodejs-buildpack/releases/tag/v1.8.4).
```

**Comment by jjtang1985** [comment]
@Joysie ,

From the cloud foundry nodes buildpack [release notes](https://github.com/cloudfoundry/nodejs-buildpack/releases/tag/v1.8.4), **node 18 is the default version**, which is not reflected by the doc here.

Therefore, this PR fixes such missing GAP and hopefully we stop confusing users with outdated docs.

Thank you.

Best regards,
Junjie

**Comment by Joysie** [comment]
Hi all,

Thank you for your feedback and proposals! 

I updated the documentation - the changes will be visible to customers with the next SAP BTP release (Jan 12). And to confirm the message to our customers here:

The SAP _nodejs_buildpack_ supports the following versions:

- Node.js 14 – supported but going out of maintenance soon
- Node.js 16 – default version
- Node.js 18 

**Caution:** For SAP HANA XSA applications, we recommend that you keep using version 14 some more due to an issue with the **@sap/xsjs** library, which is not compatible with Node.js 16 and higher. Once we provide a solution for XSA applications to run on version 16, we'll let you know via the official SAP BTP release notes.

@jjtang1985, 

The latest Node buildpack version that SAP currently provides is 1.8.3, thus - Node 16 is set as default. With the next release, when we update it to 1.8.4, I'll update the documentation again, with Node 18 as a default version.

Hope that helps!
Gergana

**Review by Joysie — APPROVED** [review]

**Comment by Joysie** [comment]
Thank you, @fwilhe 

Well, I can merge the PR right now  but I prefer to wait until changes are publicly visible on the Help Portal.

But if keeping the PR open for a week is not OK, I'll merge and close it now.

Best regards,
Gergana

**Comment by fwilhe** [comment]
Hi @Joysie 

I was not aware of the process in this repo. It's okay to keep it open if this makes sense, what's important to me is that the change is reflected in the help portal eventually.

Thanks

**Comment by nikolasimeonov77** [comment]
Hi @Joysie,
Please update the topic in our DITA CCMS and then merge the proposed change. It doesn't make sense to keep the pull request open for a week. It will be updated on the SAP Help Portal with the next release.

Thanks!

Sincerely,
Nikola

**Comment by Joysie** [comment]
OK, thanks Nikola!

Update is made. Merging the PR now.

**Comment by fwilhe** [comment]
https://people.sap.com/wilhelmflo

**Comment by Joysie** [comment]
Documentation is updated on SAP Help Portal.  See: [Developing Node.js in the Cloud Foundry Environment ](https://help.sap.com/docs/btp/sap-business-technology-platform/developing-node-js-in-cloud-foundry-environment)

**Comment by fwilhe** [comment]
> Documentation is updated on SAP Help Portal. See: [Developing Node.js in the Cloud Foundry Environment ](https://help.sap.com/docs/btp/sap-business-technology-platform/developing-node-js-in-cloud-foundry-environment)

Thank you @Joysie 👍

**Pull request merged**